lib/common/client/middleware/response/mhv_xml_html_errors.rb

```diff
@@ -5,17 +5,33 @@ module Middleware
       module Response
         class MhvXmlHtmlErrors < Faraday::Response::Middleware
           include SentryLogging
-          attr_reader :status
+          attr_reader :status, :breakers_service
+
+          def initialize(app, options = {})
+            # set the error prefix to something like 'RX' or 'SM'
+            @breakers_service = options[:breakers_service]
+            # raise ArgumentError.new('no BreakersService provided') unless @breakers_service.is_a?(Breakers::Service)
+            super(app)
+          end
 
           def on_complete(env)
             return if env.success?
             return unless env.response_headers['content-type'] =~ /\b(xml|html)/
             @status = env.status.to_i
             @body = env.body.delete('%') # strip percentages from html because Sentry uses it for interpolation
+            @header = env.response_headers.to_s.delete('%')
 
-            extra_context = { original_status: @status, original_body: @body }
+            # Logging the header as well, because it can include a Retry-After in the header if known
+            # This might be useful for improved handling with breakers
+            extra_context = { original_status: @status, original_body: @body, original_header: @header }
+            # eventually we won't need to log_message_to_sentry for 503, but lets do it for now
             log_message_to_sentry('Could not parse XML/HTML response from MHV', :warn, extra_context)
-            raise Common::Exceptions::BackendServiceException.new('VA900', response_values, @status, @body)
+            if status == 503
+              breakers_service.begin_forced_outage!
+              raise Breakers::OutageException.new(breakers_service.latest_outage, breakers_service)
```

Comment:
If you want to mull it over for a bit, I could remove the `begin_forced_outage` for now, assuming that it will have the intended affect of incrementing the heuristics, at least until such time a) the gem could support this use case, b) we can verify if MHV ever provides Retry-After in the header.